docs/design/security/unix-tmp.md

```diff
@@ -0,0 +1,45 @@
+
+# Unix temporary files
```

Comment:
I don't have a Mac to experiment with. Does anything in here differ for Mac? It seems that Macs support ACLs as well, and in some circumstances they are used in Linux as well. Should we include guidance for those? (@tmds)

Reply:
Are ACL's relevant to many Linux users, enough that (in theory) we should set them if available? Or are they mainly for special purposes.

Reply:
Mac has its own restrictions, and doesn't necessarily have a shared temp directory. Just in case, the above should probably be followed, but it may not be necessary, depending on the configuration.